test/AspNet.Security.OAuth.Providers.Tests/OAuthTests`1.cs

```diff
@@ -253,7 +253,7 @@ void ConfigureServices(IServiceCollection services)
         using (var client = server.CreateDefaultClient(LoopbackRedirectHandler))
         {
             // Act
-            using var result = await client.GetAsync("/me");
+            using var result = await client.GetAsync("/userinfo");
```

Comment:
Please revert the changes in this file. They are unnecessary as the URL exposed by the test fixture does not need to be changed.

Reply:
Thanks for the update.
The recommended changes have been made.